tmva/tmva/CMakeLists.txt

```diff
@@ -368,8 +374,9 @@ ROOT_STANDARD_LIBRARY_PACKAGE(TMVA
     Imt
     RIO
     XMLIO
-  INSTALL_OPTIONS
-    ${installoptions}
+    ${TMVA_EXTRA_DEPENDENCIES}
+  DICTIONARY_OPTIONS -writeEmptyRootPCM ${dictoptions}
+  INSTALL_OPTIONS ${installoptions}
```

Comment:
Please don't change the coding style. One thing per line. It's for better diffs when we make changes.

Reply:
I really need to look up the coding style rules :see_no_evil:

Reply:
Fixed.

Reply:
I don't think we have it anywhere yet for CMake, but most files already follow the new style, so just keeping it consistent is enough.